### PR TITLE
Add links to member table

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Hey @balena-io/docs do take a look as we might need to add new columns to this member access table, especially `settings` and renaming some columns as well. We don't have docs for fleet actions AFAIK. 

﻿Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

File: pages/learn/manage/account.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ When an application needs to be shared with more than one user, the application
 
 ### Member types
 
-| Member Type   | Add members   | Delete App  | Add/Remove device  | Device actions  | Tags | Dev Env Variables | SSH access | Push | Fleet Env Variables | Fleet actions |
+| Member Type   | [Add members][administrator]   | Delete App  | Add/Remove device  | [Device actions][device-actions]  | [Tags][tags] | [Env Variables][env-variables] | [SSH access][ssh-access] | [Push][deploy] | [Fleet Variables][fleet-variables] | Fleet actions |
```
Comment (cywang117) [inline pages/learn/manage/account.md:63]: From what I understand, a user with an Observer role can change Env Variables but not Fleet Variables. That said, the Env Variables vs Fleet Variables naming is a bit confusing based on where they're linked to. `[env-variables]:/learn/manage/serv-vars/` (https://www.balena.io/docs/learn/manage/serv-vars/) have a fleet sections, and `[fleet-variables]:/learn/manage/configuration/` (https://www.balena.io/docs/learn/manage/configuration/) do too. I'm not sure what the difference being conveyed is.

File: pages/learn/manage/account.md [hunk 2]
```diff
@@ -174,6 +174,12 @@ If you wish to delete your {{ $names.cloud.lower }} account, go to your [_Prefer
 [ssh]:/learn/manage/ssh-access
 [starter]:/learn/manage/app-types#starter
 [administrator]: /learn/manage/organizations/#managing-roles--access-in-an-organization
+[device-actions]:/learn/manage/actions/
+[env-variables]:/learn/manage/serv-vars/
+[fleet-variables]:/learn/manage/configuration/
+[ssh-access]:/learn/manage/ssh-access/
```
Comment (lurch) [inline pages/learn/manage/account.md:180]: This is the same as the `[ssh]` link just a few lines above?

File: pages/learn/manage/account.md (hunk 1 above)
Comment (vipulgupta2048) [reply pages/learn/manage/account.md:63]: @cywang117 I was confused on that one as well. 
Env variables is fine but the page it linked to is muddled up with a lot of variables. 
Fleet variables is the fleet configuration part but yes needs more accuracy, I will take a deeper look and try to separate them out.

File: pages/learn/manage/account.md (hunk 2 above)
Comment (vipulgupta2048) [reply pages/learn/manage/account.md:180]: Thanks @lurch I must have missed that one out. Will remove this and use SSH reference link only.

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

File: pages/learn/manage/account.md (hunk 1 above)
Comment (vipulgupta2048) [reply pages/learn/manage/account.md:63]: The observer is a read-only role, hence it being able to change env var only seems fishy. We should test this before making the change + with the change that is coming in regarding variables is bound to simplify this.

Comment (cywang117) [reply pages/learn/manage/account.md]: > The observer is a read-only role

Hmm. Just looked at the [member roles table](https://www.balena.io/docs/learn/manage/account/#member-types) in the docs again and it says Observers can change "Dev Env Variables" -- sorry, I was unclear. Do you know what Dev Env Variables are @vipulgupta2048 ? I'm confused about the name.

Comment (cywang117) [reply pages/learn/manage/account.md]: Dev Env Variables vs Fleet Env Variables makes me think Observers can change device-level env vars while Developers + Admins can change fleet-level, but yeah, that doesn't make sense based on Observer being a read-only role.

Comment (cywang117) [reply pages/learn/manage/account.md]: Ah. Never mind, I just saw the proposed work in https://github.com/balena-io/docs/issues/1969. I see what you're talking about with reagrds to the incoming change.

**Comment by 20k-ultra** [comment]
I'm hoping we can close https://github.com/balena-io/docs/issues/1973 and then use it in this PR as I'm really confused what Fleet action is.

**Comment by vipulgupta2048** [comment]
You got it and I agree we should document Action first

**Comment by 20k-ultra** [comment]
Should the "Fleet Actions" heading be changed to just Actions and then we can link to https://www.balena.io/docs/learn/manage/actions/#actions ?

I opened https://github.com/balena-io/docs/issues/1973 to make a doc for "Fleet actions" but didn't realize it was just referring to any action you perform on a device which is covered in the above link.

**Comment by vipulgupta2048** [comment]
@20k-ultra I agree I think Fleet actions need to become only actions as that's exactly what we have in the dashboard. I have added that bit to the renametheon issue #1969, we also need to create a settings page.

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

**Pull request merged**